tests/virt_autotest/libvirt_routed_virtual_network.pm

```diff
@@ -35,6 +35,11 @@ use version_utils 'is_sle';
 sub run_test {
     my ($self) = @_;
 
+    #Ensure that there is enough available default pool size for virtual network test
+    my $POOL = $self->get_available_pool();
+    die "Available POOL=$POOL. The SUT needs at least 60G of POOL on the xen host for the virtual network" if $POOL < 60;
```

Comment:
Suggest use record_info("Detect pool size"); assert_script_run("test $pool >= 60G"); to record test results instead of using Perl die directly. record_info and assert_script_run are test API to be employed to record your test.

Reply:
You should append "GiB" to pool size in your message output

Reply:
I agree to add `record_info("Detect pool size: $POOL");` but `assert_script_run("test $pool >= 60G");` is just wasting time and Perl conditions as well as `die()` are OK. This is minor issue tho.

Reply:
Generally speaking, die is not a very good choice for test case checkpoint writing from my POV. You can put die in a function that accepts wrongly arguments, but I think assert_script_run is better for checkpoint. And assert_script_run also conforms to the style in this ibvirt_routed_virtual_network.pm.

Reply:
Had added `record_info()`

Reply:
> Generally speaking, die is not a very good choice for test case checkpoint writing from my POV. You can put die in a function that accepts wrongly arguments, but I think assert_script_run is better for checkpoint. And assert_script_run also conforms to the style in this ibvirt_routed_virtual_network.pm.

Both `assert_script_run()` and `die()` will die. But as I said, this is minor, I'm OK with Wayne's solution too.

Reply:
great, @pdostal @waynechen55 thanks so much for your great help and suggestion. 
We had the same point here. updated to use with `assert_script_run()` and `record_info()` to call out OSD API.

Reply:
DONE

Reply:
> Generally speaking, die is not a very good choice for test case checkpoint writing from my POV. You can put die in a function that accepts wrongly arguments, but I think assert_script_run is better for checkpoint. And assert_script_run also conforms to the style in this ibvirt_routed_virtual_network.pm.

I don't get this argument. `assert_script_run` uses [croak](https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L958) which is equivalent to `die`, so I don't see where is the advantage of using `assert_script_run` over `die` for such a simple comparison. What am I missing?

Reply:
I don't want to block this PR, it's OK to go as this is a minor issue. I'm only wondering why `assert_script_run` is considered a better approach than `die` here. I'm just curious :-)